roxie/ccd/ccdserver.cpp

```diff
@@ -26450,6 +26453,7 @@ class CRoxieServerKeyedJoinBase : public CRoxieServerActivity, implements IRecor
     virtual void reset()
     {
         CRoxieServerActivity::reset();
+        joinGroupAllocator->emptyCache();
```

Comment:
Not sure what the benefit of this is. If it's in a child query, the cache would be useful. If it's not, emptying the cache here is pointless? At least based on my memory/understanding of how this works...

Reply:
This releases any rows that have been allocated and haven't been used.  Without it you will get complains about leaked rows.